lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -190,6 +189,15 @@ def set_tag(self, scope, key, value):
         )
 
 
+class SettingsService(object):
+    """
+    Allow XBlocks to access the server settings
+    """
+    def get(self, setting_name, default=None):
+        """ Gets setting value from django settings """
+        return getattr(settings, setting_name, default)
```

Comment:
There are a number of django settings that have secure information in them. Unless there's good reason not to, could the XBlock settings service be "confined" to a particular subset of the django settings?